tests/backup_restore_test.go

```diff
@@ -81,6 +81,10 @@ func TestServer_BackupAndRestore(t *testing.T) {
 			t.Fatalf("query results wrong:\n\texp: %s\n\tgot: %s", expected, res)
 		}
 
+		// it takes some time for _internal to be created, we want to make sure it's treated
+		// correctly.
+		time.Sleep(time.Second * 10)
+
```

Comment:
In order to guarantee that the `_internal` database has been created, and that your code correctly skips its shards, you could add an assertion here that the `_internal` database exists.